### PR TITLE
#9 adding the function for geospaas parameters and its test

### PR DESCRIPTION
**Pull request by aanersc**

### CODE REVIEW

File: metanorm/normalizers/sentinel1_identifier.py [hunk 1]
```diff
@@ -74,3 +74,8 @@ def get_provider(self, raw_attributes):
         """ returns the suitable provider based on the filename """
         if self.match_identifier(raw_attributes):
             return utils.get_gcmd_provider(['ESA/EO'])
+
+    def get_geospaas_parameter(self, raw_attributes):
+        """ returns the suitable provider based on the filename """
```
Comment (akorosov) [inline metanorm/normalizers/sentinel1_identifier.py:79]:
```suggestion
    def get_geospaas_parameters(self, raw_attributes):
        """ return list with sigma0 parameter from wkv variable """
```

File: tests/normalizers/test_sentinel1_identifier.py [hunk 2]
```diff
@@ -19,7 +19,8 @@ def setUp(self):
             "instrument",
             "time_coverage_start",
             "time_coverage_end",
-            "provider"
+            "provider",
+            "geospaas_parameter"
```
Comment (akorosov) [inline tests/normalizers/test_sentinel1_identifier.py:23]:
```suggestion
            "geospaas_parameters"
```

File: tests/normalizers/test_sentinel1_identifier.py [hunk 3]
```diff
@@ -101,6 +102,23 @@ def test_entry_id_identification(self):
             'S1A_EW_GRDM_1SDH_20150702T172954_20150702T173054_006635_008DA5_55D1'
         )
 
+    def test_parameter_identification(self):
```
Comment (akorosov) [inline tests/normalizers/test_sentinel1_identifier.py:105]:
```suggestion
    def test_parameters_identification(self):
```

File: tests/normalizers/test_sentinel1_identifier.py [hunk 4]
```diff
@@ -101,6 +102,23 @@ def test_entry_id_identification(self):
...
+        """ Shall return the correct sentinel1 parameter based on the filename """
+        result_normalization = self.normalizer.normalize(
+            {'Identifier': 'S1A_EW_GRDM_1SDH_20150702T172954_20150702T173054_006635_008DA5_55D1'})
+        self.assertEqual(
+            result_normalization['geospaas_parameter'],
```
Comment (akorosov) [inline tests/normalizers/test_sentinel1_identifier.py:110]:
```suggestion
            result_normalization['geospaas_parameters'],
```

**Comment by aanersc** [comment]
All in plural form in 9c9dc7b based on your suggestions.

**Pull request merged**